README.md

```diff
@@ -1,2 +1,61 @@
 # tutoring-platform
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+## Table of contents
+* [About The Project](#about-the-project)
+* [Technologies](#technologies)
+* [Getting Started](#getting-started)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+
+## About The Project
```

Comment:
Ja bym dał `the project` z małej litery, bo w sekcji `Getting started` drugi wyraz jest z małej. A samo The Project nie jest nazwą własną tego projektu.